pkg/products/marin3r/config/config.go

```diff
@@ -9,9 +9,9 @@ import (
 )
 
 const (
-	RateLimitConfigMapName = "sku-limits-managed-api-service"
+	RateLimitConfigMapName = "quota-limits-managed-api-service"
 	AlertConfigMapName     = "rate-limit-alerts"
-	ManagedApiServiceSKU   = "RHOAM SERVICE SKU"
+	ManagedApiServiceQuota = "RHOAM SERVICE QUOTA"
```

Comment:
Same here -> https://gitlab.cee.redhat.com/service/managed-tenants/-/blob/main/addons/managed-api-service/metadata/stage/sku-limits.yaml.j2#L9
We wont change these values as they can be removed once we make the switch to the dynamic quota config map